test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -20,6 +19,7 @@
 if not dist.is_available():
     print("Distributed not available, skipping tests", file=sys.stderr)
     sys.exit(0)
+from torch._inductor.utils import is_gpu
```

Comment:
Is it fine for us to depend on inductor code?

Reply:
I think it is a bit weird.